src/lwc/healthCheckGroup/healthCheckGroup.js

```diff
@@ -0,0 +1,182 @@
+        collapsedIcon: 'utility:chevronright',
+        errorIcon: 'utility:error',
+        successIcon: 'utility:success',
+    }
+
+    get healthCheckResultsId(){
+        let tempString = this.healthCheckDefinition.name + 'Results';
+        return tempString.split(" ").join("");
+    }
+
+    get collapsableIcon() {
+        if (this.isExpanded){
+            return this.iconReference.expandedIcon;
+        }
+
+        return this.iconReference.collapsedIcon;        
+    }
+
+    get collapsableIconAltText(){
+        if (this.isExpanded){
+            return this.labelReference.stgHealthCheckResultsCollapse;
+        }
+        
+        return this.labelReference.stgHealthCheckResultsExpand;
+    }
+
+    get statusIcon() {
+        if (this.isAllSuccessStatus()){
+            return this.iconReference.successIcon;
+        }
+
+        return this.iconReference.errorIcon;        
+    } 
+
+    get statusIconVariant() {
+        if (this.isAllSuccessStatus()) {
+            return 'success';
+        }
+
+        return 'error';
+    }
+
+    get passedChecksDisplay() {
+        if (this.passedChecks == this.totalChecks){
+            return this.labelReference.stgHealthChecksAllPassed;
+        }
+
+        return this.labelReference.stgHealthChecksNotAllPassed.replace(
+            '{0}', this.passedChecks
+        ).replace(
+            '{1}',this.totalChecks
+        );
+    }
+
+    isAllSuccessStatus(){
+        if (this.passedChecks == this.totalChecks){
+            return true;
+        }
+
+        return false;
+    }
+
+    toggleExpanded() {
+        this.isExpanded = !this.isExpanded; 
+    }
+
+    @wire(
+        getHealthCheckGroupViewModel, 
+        {
+            name: '$healthCheckDefinition.name', 
+            className: '$healthCheckDefinition.className', 
+            namespace: '$healthCheckDefinition.namespace'
+        })
+    treeData({ error, data }) {
+        console.log('wiring data');
+        if ( data ) {
+            let stringified = JSON.stringify(data);            
+            let tempData = JSON.parse( stringified );
+
+            this.totalChecks = tempData.totalChecks;
+            this.passedChecks = tempData.passedChecks;
+
+            this.isExpanded = !(this.isAllSuccessStatus());
+
+            this.healthCheckGroupName = tempData.label;
+
+            let tempArray = [].concat(tempData.healthCheckItemList);            
+            this.processChildren(tempArray);
+
+            this.healthCheckItemList = tempArray;
+            this.expandedRowsList = tempData.expandedRows;
+
+            this.dispatchEvent(new CustomEvent('healthcheckgrouploaded'));
+
+        } else if ( error ) {
+            if ( Array.isArray( error.body ) )
+                console.log( 'Error is ' + error.body.map( e => e.message ).join( ', ' ) );
+            else if ( typeof error.body.message === 'string' )
+                console.log( 'Error is ' + error.body.message );
```

Comment:
Let's comment this out for now and address them when we land on how we're going to address error handling.
```suggestion
                //console.log( 'Error is ' + error.body.map( e => e.message ).join( ', ' ) );
            else if ( typeof error.body.message === 'string' )
                //console.log( 'Error is ' + error.body.message );
```